soc/nordic/nrf54h/bicr/bicrgen-schema.json

```
@@ -0,0 +1,422 @@
+{
+  "title": "nRF54H20 BICR Configuration",
+  "type": "object",
+  "properties": {
+    "power": {
+      "type": "object",
+      "title": "Power supply configuration",
+      "properties": {
+        "scheme": {
+          "type": "string",
+          "title": "Power supply scheme",
+          "enumNames": [
+            "Unconfigured, system will not boot",
```

Separate statements.
```suggestion
            "Unconfigured; system will not boot",
```
Alt, to stay in line with "Not supplied (P2 not used)" later on:
```suggestion
            "Unconfigured (system will not boot)",
```